### DISCUSSION TITLE
Common Tags with Micrometer + Opentelemetry

### DISCUSSION THREAD

**Discussion by lbroderi-li** [open-telemetry/opentelemetry-java-instrumentation#9542] [Q&A]
How can you add common tags to all metrics with OpenTelemetry? I am using micrometer and have setup an OpenTelemetryMeterRegistry using the .config().commonTags(commonTags) addon. However, these tags are not appearing on my metrics, so wondering if there is a different approach recommended?

**Comment by laurit** [comment]
I tried it and it worked for me. Please provide a minimal sample application that reproduces your issue.
